src/main.rs

```diff
@@ -81,6 +81,8 @@ fn app() -> Result {
     // crates.io
     cmds.add("?crate help", crates::help);
     cmds.add("?crate query...", crates::search);
```

Comment:
for the most part i have been adding a newline between commands that dont have anything to do with each other.  It would be nice to have a newline between the `?crate` command and the `?docs` command.